pyliquibase/__init__.py

```diff
@@ -12,21 +17,31 @@
 handler.setFormatter(formatter)
 log.addHandler(handler)
 
-
 #####
 
+LAST_STABLE_KNOWN_LB_VERSION: str = "4.6.2"
+COMPATIBLE_LIQUIBASE_VERSIONS: List = ["4.6.1", "4.6.2"]
+URL_LIQUIBASE_ZIP: str = "https://github.com/liquibase/liquibase/releases/download/v{}/liquibase-{}.zip"
+URL_LIQUIBASE_RELEASES: str = "https://api.github.com/repos/liquibase/liquibase/releases"
+LIQUIBASE_ZIP_NAME: str = "liquibase-{}.zip"
+LIQUIBASE_PATH_TO_EXTRACT: str = "liquibase-{}"
+
 
 class Pyliquibase():
+    version: str = LAST_STABLE_KNOWN_LB_VERSION
```

Comment:
thic can be class variable, under __init__ ? 

    self.version: str = version